src/pages/home.tsx

```diff
@@ -30,7 +30,7 @@ export const HomePage: React.FC = () => {
     // Do an initial load of all Objects of a particular type
     useEffect(() => {
         if (token != null) {
-            client.ontology.objects.ReplaceMeObject.all().then(objects =>
+            client.ontology.objects.Country.all().then((objects: Country[]) =>
```

Comment:
@styu Going back to the error/loading state - does this means we don't really need to save it here in a "loading" state? 
e.g.
```
const [objectList, setObjectList] = useState<Country[]>([])
...
const dataOrError = client.ontology.objects.Country.all();
if (isSuccessfull(dataOrError)){
   setObjectList(dataOrError.data),
}
```

Reply:
no we do need to handle it here - my point was that the SDK itself should not handle loading states, but clients (such as React code) might have to